pkg/datasource/csvsource/csv_kline_reader_test.go

```diff
@@ -0,0 +1,155 @@
+package csvsource
+
+import (
+	"encoding/csv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+var assertKLineEq = func(t *testing.T, exp, act types.KLine) {
+	assert.Equal(t, exp.StartTime, act.StartTime)
+	assert.Equal(t, exp.Open.Compare(act.Open), 0)
+	assert.Equal(t, exp.High.Compare(act.High), 0)
+	assert.Equal(t, exp.Low.Compare(act.Low), 0)
```

Comment:
for the Equal method:
second arg is the expected value
third arg is the actual value